cmd/lncli/cmd_pay.go

```diff
@@ -405,6 +413,10 @@ func sendPaymentRequest(ctx *cli.Context,
 
 			return nil
 		}
+
+		if showInflight {
```

Comment:
Nice, this was something I was missing when I was testing things out late last week. I ended up resorting to piping the final output of `payinvoice` into a file and doing `tail -f` on `lnd`'s logs.

Reply:
Yes, there is probably still a long tail of mpp diagnostics to be expected.